Packs/AWS/Integrations/AWS/AWS.yml

```
@@ -1397,6 +1608,230 @@ script:
+        required: true
+        prettypredefined:
+          RDS Instance ID: ${asset.xdm.asset.name}
+      - name: publicly_accessible
+        description: 'Weather the DB instance is publicly accessible.'
+        defaultValue: 'false'
+        hidden: true
+      - name: apply_immediately
+        description: 'Weather the modifications in this request and any pending modifications are asynchronously applied as soon as possible, regardless of the PreferredMaintenanceWindow setting for the DB instance. By default, this parameter is disabled. If this parameter is disabled, changes to the DB instance are applied during the next maintenance window. Some parameter changes can cause an outage and are applied on the next call to RebootDBInstance , or the next failure reboot.'
+        defaultValue: 'true'
+        hidden: true
+
+  - name: aws-rds-db-instance-modify-copy-tags-on-rds-snapshot-quick-action
+    description: This action enables 'copy tags to snapshots' for the RDS instance by running the aws rds modify-db-instance command in your environment. It requires 'rds:ModifyDBInstance' permission.
+    prettyname: Enable Copy Tags on RDS Snapshot
+    hidden: true
+    quickaction: true
+    arguments:
+      - name: account_id
+        description: The AWS account ID.
+        prettyname: Account ID
+        required: true
+        prettypredefined:
+          Account ID: ${asset.xdm.asset.realm}
+      - name: region
+        description: The AWS region.
+        prettyname: Region
+        required: true
+        prettypredefined:
+          Region: ${asset.xdm.asset.cloud.region}
+      - name: db_instance_identifier
+        prettyname: RDS Instance ID
+        description: 'The identifier of DB instance to modify.'
+        required: true
+        prettypredefined:
+          RDS Instance ID: ${asset.xdm.asset.name}
+      - name: copy_tags_to_snapshot
+        description: 'Weather to copy all tags from the DB instance to snapshots of the DB instance. By default, tags aren’t copied.'
+        defaultValue: 'true'
+        hidden: true
+      - name: apply_immediately
+        description: 'Weather the modifications in this request and any pending modifications are asynchronously applied as soon as possible, regardless of the PreferredMaintenanceWindow setting for the DB instance. By default, this parameter is disabled. If this parameter is disabled, changes to the DB instance are applied during the next maintenance window. Some parameter changes can cause an outage and are applied on the next call to RebootDBInstance , or the next failure reboot.'
+        defaultValue: 'true'
+        hidden: true
+
+  - name: aws-rds-db-instance-modify-enable-automatic-backup-quick-action
+    description: Enables the automatic backup settings for the RDS dB instance with a default retention period of 30 days by running the aws rds modify-db-instance command in your environment. Successful execution requires 'rds:ModifyDBInstance' permission.
```

```suggestion
    description: Enables the automatic backup settings for the RDS dB instance with a default retention period of 30 days by running the aws rds modify-db-instance command.
```
Replace aws rds modify-db-instance.